upload-api/buckets/delegations-store.js

```diff
@@ -23,6 +23,25 @@ export function createDelegationsStore(region, bucketName, options = {}) {
   return useDelegationsStore(s3client, bucketName)
 }
 
+/**
+ * 
+ * @param {string} endpoint 
+ * @param {string} accessKeyId 
+ * @param {string} secretAccessKey 
+ * @param {string} bucketName
+ */
+export function useR2DelegationsStore(endpoint, accessKeyId, secretAccessKey, bucketName){
```

Comment:
```suggestion
export function createR2DelegationsStore(endpoint, accessKeyId, secretAccessKey, bucketName){
```

Reply:
hm, I'd like to follow the convention used by the other modules in `/buckets` where the `use*` function takes an endpoint, accessKeyId, etc and the `create*` function takes an s3 client. I've updated the name of the other function in this module to match this pattern, but left this one as-is.

Reply:
Here it is not the case right? Looking at above create + https://github.com/web3-storage/w3infra/blob/main/upload-api/buckets/car-store.js `use` receives an S3 client, while create creates one

Reply:
oh you're totally right, I was confused, thanks and good catch!